test/client-spec.js

```diff
@@ -15,18 +15,25 @@ var request = {
 var mockServerPort = config.janus.server.port;
 var mockServerUrl = config.janus.server.url;
 
+var brokenMockServerPort = config.janus.server.port + 1;
+var brokenMockServerUrl = 'http://localhost:' + brokenMockServerPort;
+
 describe('Client', function() {
 
     var janusServerMock;
+    var brokenJanusServerMock;
     before(function(done){
         janusServerMock = new JanusServerMock({
             port: mockServerPort
         });
-        janusServerMock.init().then(()=>{
-            done();
-        }).catch(()=>{
-            done(err);
+        brokenJanusServerMock = new JanusServerMock({
+            port: brokenMockServerPort,
+            triggerHandshakeTimeout: true
         });
+        Promise.all([
+            janusServerMock.init(),
+            brokenJanusServerMock.init()
+        ]).then(() => done()).catch((e) => done(e));
```

Comment:
Just realized I forgot to add the call to `brokenJanusServerMock.close()` in the after hook. Will update.